tf_files/configs/kube-setup-fence.sh

```diff
@@ -43,9 +43,15 @@ if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then # update secrets
   mkdir -p jwt-keys
   mkdir -p ssh-keys
 
-  if [ ! -f jwt-keys/jwt_public_key.pem ]; then
-    openssl genrsa -out jwt-keys/jwt_private_key.pem 2048
-    openssl rsa -in jwt-keys/jwt_private_key.pem -pubout -out jwt-keys/jwt_public_key.pem
+  # Create keypairs for fence. Following the requirements from fence, the
+  # keypairs go in subdirectories of the base keys directory, where the
+  # subdirectories are named as an ISO 8601 timestamp of when the keypair is
+  # created.
+  if [ ! -f jwt-keys/*/jwt_public_key.pem ]; then
+    timestamp=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Ugh!
My comment didn't stick for some reason.
Anyway - should change this a little to preserve existing keys, and handle the case where multiple keys exist:

```
if ! ls jwt-keys/*/jwt_public_key.pem > /dev/null 2>&1; then
  destDir=...
  mkdir -p "$destDir"
  if [ -f jwt_keys/jwt_public_key.pem && -f jwt_keys/jwt_private_key.pem ]; then
    # preserve existing keys
    mv jwt_keys/*.pem "$destDir/"
  else
    # generate new key
  ...

Reply:
I updated this, let me know if this is like what you were thinking: https://github.com/uc-cdis/cloud-automation/pull/301/files#diff-b558d4f4bd1ba4f2a00b47a1507c5173R79